src/util/templates/EjsTemplateEngine.ts

```diff
@@ -11,21 +11,27 @@ import Dict = NodeJS.Dict;
  */
 export class EjsTemplateEngine<T extends Dict<any> = Dict<any>> implements TemplateEngine<T> {
   private readonly applyTemplate: Promise<TemplateFunction>;
+  private readonly rootPath: string;
 
   /**
    * @param template - The default template @range {json}
    */
-  public constructor(template?: Template) {
+  public constructor(baseUrl: string, template?: Template) {
```

Comment:
Even though there are no templates that currently need it, I would also update the handlebars template engine similarly.